<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Source image</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PiiImageDisplay" name="_pSourceImageDisplay" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Blue channel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PiiImageDisplay" name="_pBlueChannelDisplay" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Red channel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PiiImageDisplay" name="_pRedChannelDisplay" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Green channel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PiiImageDisplay" name="_pGreenChannelDisplay" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="PushButton" name="_pPreviousImageButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PushButton" name="_pNextImageButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="_pActionSelectImages"/>
    <addaction name="separator"/>
    <addaction name="_pActionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="_pActionSelectImages">
   <property name="text">
    <string>Select images...</string>
   </property>
  </action>
  <action name="_pActionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEka">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eka</string>
   </property>
  </action>
  <action name="actionToka">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Toka</string>
   </property>
  </action>
  <action name="actionKolmas">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kolmas</string>
   </property>
  </action>
  <action name="actionNelj_s">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Nelj√§s</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PiiImageDisplay</class>
   <extends>QWidget</extends>
   <header>PiiImageDisplay.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>PushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>_pActionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
